envoy/config/metrics/v2/stats.proto

```diff
@@ -21,6 +21,8 @@ message StatsSink {
   // * :ref:`envoy.statsd <envoy_api_msg_config.metrics.v2.StatsdSink>`
   // * :ref:`envoy.dog_statsd <envoy_api_msg_config.metrics.v2.DogStatsdSink>`
   // * :ref:`envoy.metrics_service <envoy_api_msg_config.metrics.v2.MetricsServiceConfig>`
+  //
+  // Some sinks support tagged/multiple dimentions metrics and some do not.
```

Comment:
I would rephrase this as "Sinks optionally support tagged/multiple dimension metrics."